src/cap2/estruturas_de_controle.md

```diff
@@ -87,3 +87,85 @@ int main()
    return 0;
 }
 ```
+
+### As instruções `continue` e `break`
+
+Quando utilizamos um laço, é possível que queiramos interrompê-lo prematuramente. Para isso existe a instrução `break`:
+
+```cpp
+#include <iostream>
+
+int main()
+{
+    for (int i = 1; i <= 100; i = i + 1) {
+        std::cout << "Iteração #" << i << '\n'
+                  << "Você deseja continuar? [s/n]: ";
+
+        char op;
+        std::cin >> op;
+        if (op == 'n')
+            break;
+        
+        std::cout << "Incrementando i..." << std::endl;
+    }
+
+    return 0;
+}
+```
+
+Assim que o _thread_ de execução atinge a instrução `break`, o laço é imediatamente interrompido; o `i` deixa de ser incrementado e vamos direto para nosso `return`.
```

Comment:
Explain what a `thread` is, or what you mean by `thread` in this context, or change to avoid relying on reader's knowledge.

Examples:
```
Assim que a linha de execução do program atinge uma instrução `break` (...)
```

```
No momento que o programa alcança uma instrução `break` (...)
```

Reply:
Addressed